item.rb

```diff
@@ -0,0 +1,51 @@
+require_relative 'years_difference'
+
+class Item
+  include YearsDifference
+  attr_accessor :publish_date, :archived
+
+  def initialize(publish_date, archived, id = Time.now.to_f.to_s)
+    @id = id
+    @publish_date = publish_date
+    @archived = archived
+  end
+
+  def add_genre(genre)
+    @genres << genre
+  end
+
+  def add_source(source)
+    @sources << source
+  end
+
+  attr_writer :label
+
+  def add_author(author)
+    # @author << author
```

Comment:
Great job team 🤝

- [ ] Kindly remove all commented code from your project as this is regarded as a bad practice.